app/presenters/tree_builder_policy.rb

```diff
@@ -65,12 +80,18 @@ def x_get_tree_custom_kids(parent, count_only, options)
       pid = parent[:id]
 
       # Push folder node ids onto open_nodes array
-      %W(xx-#{pid}_xx-#{pid}-host xx-#{pid}_xx-#{pid}-vm xx-#{pid}_xx-#{pid}-containerImage).each { |n| open_node(n) }
+      %W(xx-#{pid}_xx-#{pid}-host
+         xx-#{pid}_xx-#{pid}-vm
+         xx-#{pid}_xx-#{pid}-containerReplicator
+         xx-#{pid}_xx-#{pid}-containerGroup
+         xx-#{pid}_xx-#{pid}-containerNode
+         xx-#{pid}_xx-#{pid}-containerImage).each { |n| open_node(n) }
```

Comment:
you should be able to iterate over UI_FOLDERS and prefix with xx-#{pid}_xx-#{pid}- inside the loop